src/main/java/nextstep/subway/favorite/application/FavoriteService.java

```diff
@@ -0,0 +1,68 @@
+package nextstep.subway.favorite.application;
+
+import nextstep.subway.ServiceException;
+import nextstep.subway.favorite.domain.Favorite;
+import nextstep.subway.favorite.domain.FavoriteRepository;
+import nextstep.subway.favorite.dto.FavoriteRequest;
+import nextstep.subway.favorite.dto.FavoriteResponse;
+import nextstep.subway.member.domain.Member;
+import nextstep.subway.member.domain.MemberRepository;
+import nextstep.subway.station.domain.Station;
+import nextstep.subway.station.domain.StationRepository;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class FavoriteService {
+  private FavoriteRepository favoriteRepository;
```

Comment:
```suggestion
  private final FavoriteRepository favoriteRepository;
```

알고계시지만 놓친 것 같네요 :)

다른곳도 확인해주시면 좋을 것 같습니다 ~

Reply:
앗 넵 확인하겠습니다!